From 22bff509e2a0e5640994bc92ded62c383d99bc37 Mon Sep 17 00:00:00 2001
From: Pierre-Jean Texier <pjtexier@koncepto.io>
Date: Mon, 24 Feb 2020 21:44:45 +0100
Subject: [PATCH] Makefile.am: add build rule for README

Fixes:

| configure.ac:17: installing 'config/config.sub'
| configure.ac:16: installing 'config/install-sh'
| configure.ac:16: installing 'config/missing'
| Makefile.am: error: required file './README' not found
| libinotifytools/src/Makefile.am: installing 'config/depcomp'

Upstream-Status: Submitted [https://github.com/inotify-tools/inotify-tools/pull/111]

Signed-off-by: Pierre-Jean Texier <pjtexier@koncepto.io>
---
 Makefile.am | 2 ++
 autogen.sh  | 1 -
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 56e4531..d08c8fe 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,6 +2,8 @@ ACLOCAL_AMFLAGS = -I m4
 
 SUBDIRS = libinotifytools src man
 
+README: README.md
+
 dist-hook:
 # Automake official documentation states that dist-hook should assume files
 # are not writable.
diff --git a/autogen.sh b/autogen.sh
index 48eb21d..31572c2 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-cp README.md README
 autoreconf --install "$@" || exit 1
-- 
2.7.4

